NICE.Identity.Management/ClientApp/src/components/FilterSuggestions/FilterSuggestions.module.scss

```diff
@@ -0,0 +1,48 @@
+@import '@nice-digital/nds-core/scss/core.scss';
+
+.suggestions {
+  position: relative;
+
+  ul {
+    list-style: none;
+    background-color: #fff;
```

Comment:
I believe there's a colour variable for white in the core....$nds-colour-nice-white instead of #fff perhaps?